docs/topics/logging.txt

```diff
@@ -398,12 +408,43 @@ configure logging using your own approach), you can set
 :setting:`LOGGING_CONFIG` to ``None``. This will disable the
 configuration process.
 
-.. note::
-    Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the
-    configuration process is disabled, not logging itself. If you
-    disable the configuration process, Django will still make logging
-    calls, falling back to whatever default logging behavior is
-    defined.
+Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the configuration
+process is disabled, not logging itself. If you disable the configuration
+process, Django will still make logging calls, falling back to whatever default
+logging behavior is defined.
+
+Here's an example of how to make the logging system print everything to the
+console without having Django's disabled default loggers swallow entries. It
+may be useful during local development. Set the environment variable
+``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging calls in the console.
+
+.. snippet::
```

Comment:
How can you get messages from the `django.request` logger while leaving Django's default configuration in place? Here is what it does:
```
        'django.request': {
            'handlers': ['mail_admins'],
            'level': 'ERROR',
            'propagate': False,
        },
```
Here is what I have in my test project:
```
LOGGING = {
    'version': 1,
    'disable_existing_loggers': False,
    'handlers': {
        'console': {
            'level': os.getenv('LOGLEVEL', 'DEBUG'),
            'class': 'logging.StreamHandler',
            'formatter': 'debugging',
        },
    },
    'formatters': {
        'debugging': {
            'format': '%(name)s %(levelname)s %(module)s:%(lineno)s: %(message)s'
        },
    },
    'root': {
        'handlers': ['console'],
        'level': 'DEBUG',
    },
    'django.request': {
        'handlers': ['console'],
        'level': 'DEBUG',
        'propagate': True,
    },
}
```